mbf_abstract_nav/include/mbf_abstract_nav/abstract_controller_execution.h

```diff
@@ -234,6 +243,9 @@ namespace mbf_abstract_nav
     //! The time the controller responded with a success output (output < 10).
     ros::Time last_valid_cmd_time_;
 
+    //The first time when the actual robot velocity is abnormal comparing to the velocity command
```

Comment:
When modifying existing code, follow the same comment and code stiles
```suggestion
    //! The first time when the actual robot velocity is abnormal comparing to the velocity command
```